### PR TITLE
docs(clustered): clarify tuning garbage collector env vars

### PR DESCRIPTION
**Pull request by jdockerty**
A lower value here means that the garbage collector can remove files which are "newer", in the sense that a value of `6h` means the parquet files only need to be older than 6 hours and marked as `to_delete` for the GC to delete them. The opposite would be true in the case of a value of `30d`.

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

**Comment by jdockerty** [comment]
cc: @domodwyer If you don't mind, can you double check my understanding here as this is for public-facing docs :smile:

**Review by domodwyer — APPROVED** [review]
It's true!

File: content/influxdb/clustered/write-data/best-practices/data-lifecycle.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ Use the following environment variables to tune the garbage collector:
 - `INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF`: how long to retain rows in the Catalog
   that reference Parquet files marked for deletion. The default is `30d`.
 
+In short, these values tune how aggressive the garbage collector can be. A lower
+value means that files can be removed at a faster pace.
```
Comment (sanderson) [inline content/influxdb/clustered/write-data/best-practices/data-lifecycle.md:49]:
```suggestion
These values tune how aggressive the garbage collector can be. A shorter duration
value means that files can be removed at a faster pace.
```

**Review by sanderson — APPROVED** [review]
Just some minor style suggestions. Looks good. Thanks @jdockerty!

**Pull request merged**